package.json

```diff
@@ -46,7 +46,7 @@
     "dist/",
     "ios/",
     "android/",
-    "CapacitorCommunityCameraPreview.podspec"
+    "COtecnaCameraPreview.podspec"
```

Comment:
please rename to CotecnaCameraPreview.podspec

Reply:
Done